x-pack/plugins/cases/public/components/links/index.tsx

```diff
@@ -16,6 +16,16 @@ import {
 import React, { useCallback } from 'react';
 import * as i18n from './translations';
 
+export interface CasesNavigation {
+  href: string;
+  onClick?: (arg: React.MouseEvent | MouseEvent) => void;
+}
+
+export interface CasesGetNavigation<T = React.MouseEvent> {
```

Comment:
I'm thinking we should make all the `href` => `getHref`, and the argument is optional `getHref: (arg?: T) => string;`. Two types seems confusing to me

Reply:
The main reason behind this was for getHref to inform that it's a function to call to get the href based on some parameters vs the href value itself.

Reply:
Though, typescript might make that distinction unnecessary though... 🤔

Reply:
I'd prefer a single type, especially if we are trying to have uniform navigation props. this is not uniform

Reply:
👍🏾 done 😄